Python/cim-convert-tool/configAcCim17.py

```diff
@@ -0,0 +1,110 @@
+configAcCim17 = \
```

Comment:
config script looks fine, but there seems to be some errors in the [DIGIN10-30-M1_AC.xml]( https://github.com/digin-energi/Grunnprofil/blob/develop/DIGIN10/Asset/DIGIN10-30-M1_AC.xml)

Manufacturer.IdentifiedObject.alias missing aliasName?
	
ProductAssetModel keeps attribute as if inherited by AssetModelCatalogueItem, but there is no such inheritance in the UML. Only an association from ProductAssetModel to AssetModelCatalogueItem.